Form/Type/VichImageType.php

```diff
@@ -90,4 +106,21 @@ public function getBlockPrefix(): string
     {
         return 'vich_image';
     }
+
+    protected function resolvePath($storageResolveMethod, $object, FormInterface $form)
```

Comment:
Can we make this method private?
Can we add a return type?

Reply:
Done